### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by VictoriaBeilsten-Edmands**
- Replace dlsio imports with scisoftpy
- Specify requirement for one of PyQt4 or PySide in README
- Add classifiers, url, description and author info to setup.py
- Add LICENSE

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
-
 from setuptools import setup, find_packages
 
+install_deps = ['numpy>=1.6', 'scisoftpy>=2.16']
+try:
+    import PySide
```
Comment (PeterC-DLS) [inline setup.py]: If this succeeds then add it as a dependency

**Review by PeterC-DLS — APPROVED** [review]

**Review by PeterC-DLS — CHANGES_REQUESTED** [review]
Just a change in setup.py needed

**Comment by VictoriaBeilsten-Edmands** [comment]
PySide/ PyQt4 requirement removed from setup.py, but added a line in README to state they're required.

File: README [hunk 2]
```diff
@@ -2,3 +2,6 @@ This repository contains ScisoftPy scripts for processing high resolution powder
 diffraction (HRPD) data.
 
 They are aimed for use with I11's data files to sum and re-bin overlapping scans.
+
+Additional dependencies:
+One of PySide>=1.2 or PyQt4>=4 is required.
```
Comment (PeterC-DLS) [inline README:7]: Also need to print error messages when it occurs during import in `mainui.py`

**Review by PeterC-DLS — CHANGES_REQUESTED** [review]
As per comment on README

**Review by PeterC-DLS — APPROVED** [review]

**Pull request merged**